Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -69,6 +69,7 @@
 --rules spaceAroundParens
 --rules enumNamespaces
 --rules blockComments
+--rules docComments
```

Comment:
This will fail in CI until we pull in a version of SwiftFormat that includes this rule implementation